src/components/modals/PrivacySettingsModal.vue

```diff
@@ -72,7 +102,13 @@
             <span class="close" v-on:click="removeWhitelistedAddress(address)">×</span>
           </div>
         </div>
-        <div v-else>
+        <div v-if="whitelistedEmails.length > 0">
+          <div v-for="email in whitelistedEmails" :key="email">
+            <DisplayName :name="email" />
+            <span class="close" v-on:click="removeWhitelistedEmail(email)">×</span>
```

Comment:
Use the `@` short-hand instead of `v-on`:

```suggestion
            <span class="close" @click="removeWhitelistedEmail(email)">×</span>
```

Reply:
Does Bootstrap come with an icon X? Feels weird to use the character `x` instead

Reply:
I'm not really sure. Probably? But I don't think we use fontawesome which I think is what bootstrap would need to use it's icons. I dunno, I just copied what was already there.